aws/requests.yaml

```diff
@@ -2,9 +2,11 @@ releases:
 - name: ">= 20.0.0"
   requests:
   - name: observability-bundle
-    version: ">= 1.3.0"
+    version: ">= 1.3.2"
```

Comment:
please add this to a "> 20.0.0" or ">= 20.0.1" section instead

Reply:
done :)